src/dawn/CodeGen/CXXNaive/ASTStencilBody.h

```diff
@@ -0,0 +1,120 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                          _
+//                         | |
+//                       __| | __ ___      ___ ___
+//                      / _` |/ _` \ \ /\ / / '_  |
+//                     | (_| | (_| |\ V  V /| | | |
+//                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
I agree, we should be consistent. What are the issues with pragma once? 
The reason why I always prefer it is, if the #ifdef guard that we put in certain header is not unique, the error is very difficult to catch. You know something is messed up loading the headers, but no  idea why, and I have spent some time debugging this in the past. Typical mistake when copy pasting headers

Reply:
I would also prefer to use pragma once for the reasons you gave. Here is C&P from [stackoverflow](https://stackoverflow.com/questions/1143936/pragma-once-vs-include-guards)

If your `#include` search path is sufficiently complicated, the compiler may be unable to tell the difference between two headers with the same basename (e.g. a/foo.h and b/foo.h), so a `#pragma once` in one of them will suppress both. It may also be unable to tell that two different relative includes (e.g. `#include "foo.h"` and `#include "../a/foo.h"` refer to the same file, so `#pragma once` will fail to suppress a redundant include when it should have.

I think that the issue outweighs the advantages (beside the fact that, in oder to be consistent, we would also need to change all the old headers) but am open to have my mind changed

Reply:
alright, fine then. You have convinced me.